app/uk/gov/hmrc/directdebitupdateemailfrontend/views/Layout.scala.html

```diff
@@ -33,17 +35,35 @@
         hmrcScripts: HmrcScripts,
         hmrcLanguageSelectHelper: HmrcLanguageSelectHelper,
         hmrcReportTechnicalIssueHelper: HmrcReportTechnicalIssueHelper,
-        requestSupport: RequestSupport
+        hmrcTimeoutDialogHelper: HmrcTimeoutDialogHelper,
+        requestSupport: RequestSupport,
+        appConfig: AppConfig
 )
-@(pageTitle: String,
+@(
+        pageTitle: String,
         hasErrors: Boolean = false,
         showBackLink: Boolean = true,
-        backLinkUrlOverride: Option[String] = None
+        backLinkUrlOverride: Option[String] = None,
+        showSignOutOverride: Option[Boolean] = None
```

Comment:
do we need this override? Is it used anywhere?

Reply:
Probably not, put it in incase, but I'll remove

Reply:
done